pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -24,27 +24,29 @@ import (
 	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/rest"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	"k8s.io/apimachinery/pkg/runtime"
 
-	"antrea.io/antrea/pkg/antctl/raw"
-	multiclusterscheme "antrea.io/antrea/pkg/antctl/raw/multicluster/scheme"
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
 )
 
-type accessTokenOptions struct {
+const (
+	createByAntctlAnnotation = "multicluster.antrea.io/created-by-antctl"
+)
+
+type memberTokenOptions struct {
 	namespace      string
 	serviceAccount string
 	roleBinding    string
 }
 
-var accessTokenOpts *accessTokenOptions
+var memberTokenOpts *memberTokenOptions
 
-var accessTokenExamples = strings.Trim(`
-# Create an access token in a leader cluster for one or more member clusters. If the ServiceAccount or RoleBinding does not exist, the command will create them too
-  $ antctl mc create accesstoken <NAME> -n <NAMESPACE> --service-account <SERVICE_ACCOUNT> --role-binding <ROLE_BINDING>
+var memberTokenExamples = strings.Trim(`
+# Create an member token in the leader cluster for one or more member clusters.
+  $ antctl mc create membertoken member-access-token -n antrea-multicluster --service-account test-sa --role-binding test-rb
```

Comment:
How about changing "member-access-token", "test-sa", "test-rb" to "cluster-east-token"?

I actually feel we can remove "--service-account" and "--role-binding" - why ask for so many names for resources created for the same token?

Reply:
Reused the token name. And now the sa and the rolebinding's name is optional.